causalpy/tests/test_pymc_experiments.py

```diff
@@ -63,7 +63,7 @@ def test_regression_kink_gradient_change():
 
 def test_inverse_prop():
     df = cp.load_data("nhefs")
-    sample_kwargs = {"tune": 100, "draws": 100, "chains": 2, "cores": 2}
+    sample_kwargs = {"tune": 100, "draws": 500, "chains": 2, "cores": 2}
```

Comment:
Could add a `random_seed` kwarg to `sample_kwargs` in order to make test output deterministic. 

So far this is not generally done in tests because we don't test for specific results, but it might be relevant here if you're getting numerical stability issues.

We do do it however in the docstring examples (under test by `doctest`) because there we do examine numerical results.

I'll leave it up to you, but this might be the more defensive play.

Reply:
Updated. Added Random seed